### DISCUSSION TITLE
Is it possible to visualize the chemical structure in WikiPathways?

### DISCUSSION THREAD

**Discussion by kozo2** [wikipathways/wikipathways-help#17] [Q&A] [answered]
I would like to create a pathway in WikiPathways like the link below.
https://www.genome.jp/pathway/map01010

Let me know if there's a way to create it.

**Comment by AlexanderPico** [comment]
We have made demos in the past to show it was possible, but we never put it into production. Can you share your use case to help us assess the priority of the feature?

**Reply by kozo2** [reply]
Yes, I think I can.
I will get back to you here when I have created a suitable pathway for it.

**Comment by DeniseSl22** [comment, marked as answer]
@kozo2 : There is a nice Cytoscape app for this, ChemViz2: https://apps.cytoscape.org/apps/chemviz2 . We cannot do this yet in PathVisio, so the Cytoscape visualization is a workaround.

**Reply by kozo2** [reply]
Thanks for the info. @DeniseSl22 
I understand. I will try to do that first with Cytoscape and ChemViz2.
